### PR TITLE
Api/221 account coins endpoint

### PR DESCRIPTION
**Pull request by stuartkuentzel**
# Description

Updates the openApi.json spec to 1.4.8.
This moves the `coins` object from `account/balance` to `account/coins`.
In reference to issue #221. The 1.4.8 spec moves the `coins` object from `account/balance` to `account/coins`.
Account Balance Response -> https://www.rosetta-api.org/docs/models/AccountBalanceResponse.html
Account Coins Response -> https://www.rosetta-api.org/docs/models/AccountCoinsResponse.html
Docs for both -> https://www.rosetta-api.org/docs/AccountApi.html#accountbalance

# Proposed Solution

- Update the openApi.json spec to 1.4.8.
- Create new `account/coins` endpoint
- Remove `coins` from `account/balance` endpoint

# Important Changes Introduced
Potentially breaking changes for users of the library, since UTXOs `coins` is moved from `account/balance` to `account/coins`.

# Testing
Adds `account-coins-api.test.ts` to `test/e2e/account`.

### CODE REVIEW

**Review by AlanVerbner — CHANGES_REQUESTED** [review]
Thanks a lot for your PR @stuartkuentzel! it looks really good. I think some test are failing tho.

Also, keep in mind that we are working on `master-next` and we are going to introduce several changes that need to happen before this PR.

File: cardano-rosetta-server/src/server/openApi.json [hunk 1]
```diff
@@ -1,1741 +1,2409 @@
...
+        "description":"The ConstructionMetadataResponse returns network-specific metadata used for transaction construction. Optionally, the implementer can return the suggested fee associated with the transaction being constructed. The caller may use this info to adjust the intent of the transaction or to create a transaction with a different account that can pay the suggested fee. Suggested fee is an array in case fee payment must occur in multiple currencies.",
+        "type":"object",
+        "required": [
+          "metadata"
+         ],
         "properties": {
           "metadata": {
-            "type": "object",
-            "properties": {
-              "ttl": { "type": "string" }
```
Comment (AlanVerbner) [inline cardano-rosetta-server/src/server/openApi.json:1441]: please, don't remove our custom fields or `fastify` will not reder them when serializing the response

**Comment by stuartkuentzel** [comment]
@AlanVerbner thanks for taking a look at it. Sounds like some big code changes will happen before this could get merged, so I'll just close it out for now. Looking forward to the updates.

**Comment by AlanVerbner** [comment]
Thanksa lot @stuartkuentzel ! I'll try to keep you posted

**Pull request closed without merging**